### ISSUE TITLE
Encrypt->Base64 indexes out of bounds for some arguments

### ISSUE THREAD

**Issue by MCRusher** [objeck/objeck-lang#16]
Using version 5.5.4 for windows x86, this test program
```
use Encryption;

class Base64Program {
	function : Main(args: String[]) ~ Nil {
		s1 := "string";
		s1->PrintLine();
		Encrypt->Base64(s1)->PrintLine();
		
		s2 := "hello";
		s2->PrintLine();
		Encrypt->Base64(s2)->PrintLine();
		
		s3 := "poop";
		s3->PrintLine();
		Encrypt->Base64(s3)->PrintLine();
	}
}
```
results in
```
string
c3RyaW5n
hello
aGVsbG8=
poop
>>> Index out of bounds in native JIT code! <<<
Unwinding local stack (00327D70):
  method: pos=2, name='Encryption.Encrypt->Base64(p0:System.String)'
  method: pos=1, name='Base64Program->Main(p0:System.String[])'
  ...
```

Side note: I like this project a lot. Feels sorta like a portable, mini java at like 1/8th the size.

**Comment by objeck** [comment]
Thanks, this has been confirmed. 

The issue seems to be in the base64 C code that was [ported](https://github.com/objeck/objeck-lang/blob/master/core/compiler/lib_src/encrypt.obs#L124). I will resolve this in a point release.

**Comment by objeck** [comment]
Fixed and released in v5.5.5
